### PR TITLE
CLOUDP-68337: internal/cli/atlas/metrics and internal/cli/opsmanager/metrics to use the simplified output

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-68337

Closes #[issue number]

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

```
./bin/mongocli om metric processes ad00123ce121da3bdc7784fdaa5dbf0b --granularity PT30M --period P1DT12H -P ops -o ""
NAME					UNITS		TIMESTAMP			VALUE
PROCESS_CPU_USER			PERCENT		2020-07-29T16:33:23Z		0.5280867
PROCESS_CPU_USER			PERCENT		2020-07-29T17:03:23Z		0.5393118
PROCESS_CPU_USER			PERCENT		2020-07-29T17:33:23Z		0.541945
PROCESS_CPU_USER			PERCENT		2020-07-29T18:33:23Z		0.0053553963
PROCESS_CPU_USER			PERCENT		2020-07-29T19:03:23Z		0.40095562
PROCESS_CPU_USER			PERCENT		2020-07-29T20:03:23Z		0.0018521702
PROCESS_CPU_USER			PERCENT		2020-07-29T21:03:23Z		0.00400066
PROCESS_CPU_USER			PERCENT		2020-07-29T21:33:23Z		0.5555369
PROCESS_CPU_USER			PERCENT		2020-07-29T22:03:23Z		0.2864721
PROCESS_CPU_USER			PERCENT		2020-07-30T05:33:23Z		0.21684264
PROCESS_CPU_USER			PERCENT		2020-07-30T06:03:23Z		0.005284037
PROCESS_CPU_USER			PERCENT		2020-07-30T07:33:23Z		0.512581
PROCESS_CPU_USER			PERCENT		2020-07-30T08:03:23Z		0.54988873
PROCESS_CPU_USER			PERCENT		2020-07-30T08:33:23Z		0.5720924
PROCESS_CPU_USER			PERCENT		2020-07-30T09:03:23Z		0.5537544
PROCESS_CPU_USER			PERCENT		2020-07-30T09:33:23Z		0.55516636
PROCESS_CPU_USER			PERCENT		2020-07-30T10:03:23Z		0.544323
PROCESS_CPU_USER			PERCENT		2020-07-30T10:33:23Z		0.54767776
PROCESS_CPU_USER			PERCENT		2020-07-30T11:03:23Z		0.542094
PROCESS_CPU_KERNEL			PERCENT		2020-07-29T16:33:23Z		0.24537982
PROCESS_CPU_KERNEL			PERCENT		2020-07-29T17:03:23Z		0.2577142
PROCESS_CPU_KERNEL			PERCENT		2020-07-29T17:33:23Z		0.26936373
PROCESS_CPU_KERNEL			PERCENT		2020-07-29T18:33:23Z		0.0029211252
PROCESS_CPU_KERNEL			PERCENT		2020-07-29T19:03:23Z		0.20047781
PROCESS_CPU_KERNEL			PERCENT		2020-07-29T20:03:23Z		0.0009260851
PROCESS_CPU_KERNEL			PERCENT		2020-07-29T21:03:23Z		0.0018464584
PROCESS_CPU_KERNEL			PERCENT		2020-07-29T21:33:23Z		0.30558184
PROCESS_CPU_KERNEL			PERCENT		2020-07-29T22:03:23Z		0.15862548
PROCESS_CPU_KERNEL			PERCENT		2020-07-30T05:33:23Z		0.10851378
PROCESS_CPU_KERNEL			PERCENT		2020-07-30T06:03:23Z		0.006605046
PROCESS_CPU_KERNEL			PERCENT		2020-07-30T07:33:23Z		0.24691841
PROCESS_CPU_KERNEL			PERCENT		2020-07-30T08:03:23Z		0.26339957
PROCESS_CPU_KERNEL			PERCENT		2020-07-30T08:33:23Z		0.28160265
PROCESS_CPU_KERNEL			PERCENT		2020-07-30T09:03:23Z		0.27271134
PROCESS_CPU_KERNEL			PERCENT		2020-07-30T09:33:23Z		0.2777981
PROCESS_CPU_KERNEL			PERCENT		2020-07-30T10:03:23Z		0.26605174
PROCESS_CPU_KERNEL			PERCENT		2020-07-30T10:33:23Z		0.26828426
PROCESS_CPU_KERNEL			PERCENT		2020-07-30T11:03:23Z		0.26327088
PROCESS_CPU_CHILDREN_USER		PERCENT		2020-07-29T16:33:23Z		N/A
PROCESS_CPU_CHILDREN_USER		PERCENT		2020-07-29T17:03:23Z		N/A
PROCESS_CPU_CHILDREN_USER		PERCENT		2020-07-29T17:33:23Z		N/A
```

```
./bin/mongocli om metric disk list  ad00123ce121da3bdc7784fdaa5dbf0b -P ops -o ""
test
```

```
./bin/mongocli om metric database list  ad00123ce121da3bdc7784fdaa5dbf0b -P ops -o ""
local
```

```
/bin/mongocli om metric disk describe ad00123ce121da3bdc7784fdaa5dbf0b test --granularity PT30M --period P1DT12H -P ops -o ""
NAME					UNITS			TIMESTAMP			VALUE
DISK_PARTITION_IOPS_READ		SCALAR_PER_SECOND	2020-07-29T16:34:23Z		N/A
DISK_PARTITION_IOPS_READ		SCALAR_PER_SECOND	2020-07-29T17:04:23Z		N/A
DISK_PARTITION_IOPS_READ		SCALAR_PER_SECOND	2020-07-29T17:34:23Z		N/A
DISK_PARTITION_IOPS_WRITE		SCALAR_PER_SECOND	2020-07-29T20:04:23Z		N/A
DISK_PARTITION_IOPS_WRITE		SCALAR_PER_SECOND	2020-07-29T21:04:23Z		N/A
DISK_PARTITION_IOPS_WRITE		SCALAR_PER_SECOND	2020-07-29T21:34:23Z		N/A
DISK_PARTITION_IOPS_WRITE		SCALAR_PER_SECOND	2020-07-29T22:04:23Z		N/A
DISK_PARTITION_IOPS_WRITE		SCALAR_PER_SECOND	2020-07-30T05:34:23Z		N/A
DISK_PARTITION_IOPS_WRITE		SCALAR_PER_SECOND	2020-07-30T06:04:23Z		N/A
DISK_PARTITION_IOPS_WRITE		SCALAR_PER_SECOND	2020-07-30T07:34:23Z		N/A
DISK_PARTITION_IOPS_WRITE		SCALAR_PER_SECOND	2020-07-30T08:04:23Z		N/A
DISK_PARTITION_IOPS_WRITE		SCALAR_PER_SECOND	2020-07-30T08:34:23Z		N/A
DISK_PARTITION_IOPS_WRITE		SCALAR_PER_SECOND	2020-07-30T09:04:23Z		N/A
DISK_PARTITION_IOPS_WRITE		SCALAR_PER_SECOND	2020-07-30T09:34:23Z		N/A
DISK_PARTITION_IOPS_WRITE		SCALAR_PER_SECOND	2020-07-30T10:04:23Z		N/A
DISK_PARTITION_IOPS_WRITE		SCALAR_PER_SECOND	2020-07-30T10:34:23Z		N/A
DISK_PARTITION_IOPS_WRITE		SCALAR_PER_SECOND	2020-07-30T11:04:23Z		N/A
DISK_PARTITION_IOPS_TOTAL		SCALAR_PER_SECOND	2020-07-29T16:34:23Z		N/A
DISK_PARTITION_IOPS_TOTAL		SCALAR_PER_SECOND	2020-07-29T17:04:23Z		N/A
DISK_PARTITION_IOPS_TOTAL		SCALAR_PER_SECOND	2020-07-29T17:34:23Z		N/A
DISK_PARTITION_IOPS_TOTAL		SCALAR_PER_SECOND	2020-07-29T18:34:23Z		N/A
DISK_PARTITION_IOPS_TOTAL		SCALAR_PER_SECOND	2020-07-29T19:04:23Z		N/A
DISK_PARTITION_IOPS_TOTAL		SCALAR_PER_SECOND	2020-07-29T20:04:23Z		N/A
DISK_PARTITION_IOPS_TOTAL		SCALAR_PER_SECOND	2020-07-29T21:04:23Z		N/A
DISK_PARTITION_IOPS_TOTAL		SCALAR_PER_SECOND	2020-07-29T21:34:23Z		N/A
DISK_PARTITION_IOPS_TOTAL		SCALAR_PER_SECOND	2020-07-29T22:04:23Z		N/A
DISK_PARTITION_IOPS_TOTAL		SCALAR_PER_SECOND	2020-07-30T05:34:23Z		N/A
DISK_PARTITION_IOPS_TOTAL		SCALAR_PER_SECOND	2020-07-30T06:04:23Z		N/A
DISK_PARTITION_IOPS_TOTAL		SCALAR_PER_SECOND	2020-07-30T07:34:23Z		N/A
DISK_PARTITION_IOPS_TOTAL		SCALAR_PER_SECOND	2020-07-30T08:04:23Z		N/A
DISK_PARTITION_IOPS_TOTAL		SCALAR_PER_SECOND	2020-07-30T08:34:23Z		N/A
DISK_PARTITION_IOPS_TOTAL		SCALAR_PER_SECOND	2020-07-30T09:04:23Z		N/A
DISK_PARTITION_IOPS_TOTAL		SCALAR_PER_SECOND	2020-07-30T09:34:23Z		N/A
DISK_PARTITION_IOPS_TOTAL		SCALAR_PER_SECOND	2020-07-30T10:04:23Z		N/A
DISK_PARTITION_IOPS_TOTAL		SCALAR_PER_SECOND	2020-07-30T10:34:23Z		N/A
DISK_PARTITION_IOPS_TOTAL		SCALAR_PER_SECOND	2020-07-30T11:04:23Z		N/A
DISK_PARTITION_UTILIZATION		PERCENT			2020-07-29T16:34:23Z		N/A
DISK_PARTITION_UTILIZATION		PERCENT			2020-07-29T17:04:23Z		N/A
DISK_PARTITION_UTILIZATION		PERCENT			2020-07-29T17:34:23Z		N/A
DISK_PARTITION_UTILIZATION		PERCENT			2020-07-29T18:34:23Z		N/A
DISK_PARTITION_UTILIZATION		PERCENT			2020-07-29T19:04:23Z		N/A
DISK_PARTITION_UTILIZATION		PERCENT			2020-07-29T20:04:23Z		N/A
DISK_PARTITION_UTILIZATION		PERCENT			2020-07-29T21:04:23Z		N/A
DISK_PARTITION_UTILIZATION		PERCENT			2020-07-29T21:34:23Z		N/A
DISK_PARTITION_UTILIZATION		PERCENT			2020-07-29T22:04:23Z		N/A

```

### CODE REVIEW

File: internal/cli/opsmanager/metrics/databases_list.go [hunk 1]
```diff
@@ -38,6 +38,10 @@ func (opts *DatabasesListsOpts) initStore() error {
 	return err
 }
 
+var DatabasesListTemplate = `NAME{{range .Results}}
```
Comment (gssbzn) [inline internal/cli/opsmanager/metrics/databases_list.go:41]: I'd say we can do without the title here, rule of thump here, if we have a single column and is easy to understand what we are returning we can do without a title

Comment (gssbzn) [reply internal/cli/opsmanager/metrics/databases_list.go:41]: Also let's not export this, `databasesListTemplate`

File: internal/cli/opsmanager/metrics/disks_describe.go [hunk 2]
```diff
@@ -39,14 +39,19 @@ func (opts *DisksDescribeOpts) initStore() error {
 	return err
 }
 
+var diskMetricTemplate = `NAME	UNITS	TIMESTAMP		VALUE{{range .ProcessMeasurements.Measurements}}
+{{.Name}}	{{.Units}}{{range .DataPoints}}	
+		{{.Timestamp}}	{{.Value}}{{end}}{{end}}
```
Comment (gssbzn) [inline internal/cli/opsmanager/metrics/disks_describe.go:44]: I left this for last as I knew it was going to be a challenge to make sense of it, and I was right, in this current format it's really hard to understand what's going on, this template may need some extra logic to it, here a couple of points for it (I'm still learning how templates work so I may type errors here)

You can use variables and `if` conditions so I'd say we need to do those here with something like
```suggestion
var diskMetricTemplate = `NAME	UNITS	TIMESTAMP		VALUE{{range .ProcessMeasurements.Measurements}} {{ $name = .Name}}	{{$unit = .Units}}
{{- if .DataPoints }}}{{range .DataPoints}}	
 {{ $name }} {{$unit}} {{.Timestamp}}	{{.Value}}{{end}}{{end}}{{end}}
```
This way each data point always has a name and a unit, we also make sure that if there's no data point we don't print anything (if of empty slices is false on templates)

File: internal/cli/opsmanager/metrics/process.go [hunk 3]
```diff
@@ -38,14 +38,19 @@ func (opts *ProcessOpts) initStore() error {
 	return err
 }
 
+var metricTemplate = `NAME	UNITS			TIMESTAMP		VALUE{{range .Measurements}}
+{{.Name}}	{{.Units}}{{range .DataPoints}}	
+			{{.Timestamp}}	{{.Value}}{{end}}{{end}}
+`
```
Comment (gssbzn) [inline internal/cli/opsmanager/metrics/process.go:44]: Similar comment to above, to make this more readable we should have a name and unit along the values

**Comment by andreaangiolillo** [comment]
This is the json of disk describe

```

./bin/mongocli om metric disk describe ad00123ce121da3bdc7784fdaa5dbf0b test --granularity PT30M --period P1DT12H -P ops
{
  "end": "2020-07-30T09:16:31Z",
  "granularity": "PT30M",
  "groupId": "5f1ee88a1bb29926644c2f2e",
  "hostId": "ad00123ce121da3bdc7784fdaa5dbf0b",
  "links": [
    {
      "rel": "self",
      "href": "http://localhost:9080/api/public/v1.0/groups/5f1ee88a1bb29926644c2f2e/hosts/ad00123ce121da3bdc7784fdaa5dbf0b/disks/test/measurements?granularity=PT30M\u0026period=P1DT12H"
    },
    {
      "rel": "http://mms.mongodb.com/host",
      "href": "http://localhost:9080/api/public/v1.0/groups/5f1ee88a1bb29926644c2f2e/hosts/ad00123ce121da3bdc7784fdaa5dbf0b"
    },
    {
      "rel": "http://mms.mongodb.com/group",
      "href": "http://localhost:9080/api/public/v1.0/groups/5f1ee88a1bb29926644c2f2e"
    }
  ],
  "measurements": [
    {
      "dataPoints": [
        {
          "timestamp": "2020-07-29T16:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T16:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T17:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T17:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T18:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T19:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T20:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T21:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T21:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T22:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T05:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T07:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T08:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T08:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T09:16:31Z",
          "value": null
        }
      ],
      "name": "DISK_PARTITION_IOPS_READ",
      "units": "SCALAR_PER_SECOND"
    },
    {
      "dataPoints": [
        {
          "timestamp": "2020-07-29T16:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T16:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T17:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T17:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T18:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T19:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T20:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T21:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T21:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T22:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T05:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T07:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T08:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T08:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T09:16:31Z",
          "value": null
        }
      ],
      "name": "DISK_PARTITION_IOPS_WRITE",
      "units": "SCALAR_PER_SECOND"
    },
    {
      "dataPoints": [
        {
          "timestamp": "2020-07-29T16:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T16:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T17:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T17:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T18:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T19:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T20:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T21:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T21:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T22:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T05:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T07:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T08:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T08:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T09:16:31Z",
          "value": null
        }
      ],
      "name": "DISK_PARTITION_IOPS_TOTAL",
      "units": "SCALAR_PER_SECOND"
    },
    {
      "dataPoints": [
        {
          "timestamp": "2020-07-29T16:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T16:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T17:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T17:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T18:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T19:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T20:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T21:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T21:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T22:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T05:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T07:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T08:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T08:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T09:16:31Z",
          "value": null
        }
      ],
      "name": "DISK_PARTITION_UTILIZATION",
      "units": "PERCENT"
    },
    {
      "dataPoints": [
        {
          "timestamp": "2020-07-29T16:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T16:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T17:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T17:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T18:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T19:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T20:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T21:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T21:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T22:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T05:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T07:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T08:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T08:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T09:16:31Z",
          "value": null
        }
      ],
      "name": "DISK_PARTITION_LATENCY_READ",
      "units": "MILLISECONDS"
    },
    {
      "dataPoints": [
        {
          "timestamp": "2020-07-29T16:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T16:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T17:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T17:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T18:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T19:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T20:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T21:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T21:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T22:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T05:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T07:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T08:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T08:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T09:16:31Z",
          "value": null
        }
      ],
      "name": "DISK_PARTITION_LATENCY_WRITE",
      "units": "MILLISECONDS"
    },
    {
      "dataPoints": [
        {
          "timestamp": "2020-07-29T16:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T16:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T17:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T17:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T18:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T19:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T20:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T21:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T21:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T22:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T05:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T07:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T08:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T08:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T09:16:31Z",
          "value": null
        }
      ],
      "name": "DISK_PARTITION_SPACE_FREE",
      "units": "BYTES"
    },
    {
      "dataPoints": [
        {
          "timestamp": "2020-07-29T16:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T16:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T17:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T17:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T18:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T19:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T20:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T21:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T21:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T22:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T05:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T07:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T08:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T08:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T09:16:31Z",
          "value": null
        }
      ],
      "name": "DISK_PARTITION_SPACE_USED",
      "units": "BYTES"
    },
    {
      "dataPoints": [
        {
          "timestamp": "2020-07-29T16:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T16:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T17:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T17:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T18:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T19:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T20:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T21:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T21:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T22:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T05:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T07:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T08:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T08:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T09:16:31Z",
          "value": null
        }
      ],
      "name": "DISK_PARTITION_SPACE_PERCENT_FREE",
      "units": "PERCENT"
    },
    {
      "dataPoints": [
        {
          "timestamp": "2020-07-29T16:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T16:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T17:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T17:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T18:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T19:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T20:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T21:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T21:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-29T22:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T05:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T07:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T08:16:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T08:46:31Z",
          "value": null
        },
        {
          "timestamp": "2020-07-30T09:16:31Z",
          "value": null
        }
      ],
      "name": "DISK_PARTITION_SPACE_PERCENT_USED",
      "units": "PERCENT"
    }
  ],
  "processId": "andreaMac:10017",
  "start": "2020-07-29T16:16:31Z",
  "partitionName": "test"
}
```

**Comment by gssbzn** [comment]
@andreaangiolillo

expected

one suggestion for the <nil>, you can do `{{if .Value }}{{ .Value }}{{else}} N/A {{end}}`

File: internal/cli/opsmanager/metrics/disks_describe.go [hunk 4]
```diff
@@ -39,14 +39,19 @@ func (opts *DisksDescribeOpts) initStore() error {
 	return err
 }
 
+var diskMetricTemplate = `NAME	UNITS	TIMESTAMP		VALUE{{range .ProcessMeasurements.Measurements}}  {{if .DataPoints}}
+{{ $name := .Name }}	{{ $unit := .Units }}{{range .DataPoints}}
```
Comment (gssbzn) [inline internal/cli/opsmanager/metrics/disks_describe.go:43]: you have an empty line every now and them due to this line, you can move it to the line above or do something like
```suggestion
{{- $name := .Name }}{{- $unit := .Units }}{{- range .DataPoints}}	
```
the dash(-) removes the code block as a space

File: internal/cli/opsmanager/metrics/process.go [hunk 5]
```diff
@@ -38,14 +38,19 @@ func (opts *ProcessOpts) initStore() error {
 	return err
 }
 
+var metricTemplate = `NAME	UNITS	TIMESTAMP		VALUE{{range .Measurements}} {{if .DataPoints}}
+{{ $name := .Name }}	{{ $unit := .Units }} {{range .DataPoints}}
```
Comment (gssbzn) [inline internal/cli/opsmanager/metrics/process.go:42]: same as above

**Comment by andreaangiolillo** [comment]
I like this way better too, it is definitely more clear than before 🚀

**Comment by gssbzn** [comment]
evergreen retry

**Comment by gssbzn** [comment]
evergreen retry

**Pull request merged**